starter-code/Driver.java

```diff
@@ -30,13 +31,124 @@ public static void main(String[] args) {
 		//Change the name of the 3rd entry to "Joe Montana"
 		
 		//Remove the 10th element
+
+		/** Answers:
+		 *
+		 1. For deleting of the first instance of Student with a lastName of "Smith":
+		      The faster one would be for the array list since it doesn't have to traverse
+		      the items that come before each element checked in each run of the loop.
```

Comment:
Deleting is faster with a Linked List, because you just change one pointer, and you don't have to shift every subsequent element over one like in an Array List